#
# Ubuntu: https://bugs.launchpad.net/bugs/334136
# Upstream: https://bugs.kde.org/show_bug.cgi?id=172217
# Description: save received files to $HOME and not $HOME/.kbluetooth4
#
--- a/src/trayicon.cpp
+++ b/src/trayicon.cpp
@@ -643,7 +643,7 @@ void KBlueTray::slotPropertyChanged(cons
 
 void KBlueTray::slotServerReady() {
 	if(!server->isStarted())
-		server->start(QDir::homePath()+"/.kbluetooth4", true, false);
+		server->start(QDir::homePath(), true, false);
 }
 
 void KBlueTray::slotServerErrorOccured(const QString& error_name, const QString& error_message) {
